Control coverage/README.md

```diff
@@ -59,3 +59,46 @@ Following controls in AzTS are currently externally scanned:
+| Azure_VirtualMachine_NetSec_Dont_Open_Restricted_Ports | Management ports must not be open on machines | Do not leave restricted ports open on Virtual Machines |
+| Azure_VNet_NetSec_Configure_NSG | Associate Subnets with a Network Security Group | NSG should be used for subnets in a virtual network to permit traffic only on required inbound/outbound ports. NSGs should not have a rule to allow any-to-any traffic |
+| Azure_Subscription_AuthZ_Remove_Deprecated_Accounts | Remove Orphaned accounts from your subscription(s) | Deprecated/stale accounts must not be present on the subscription |
+| Azure_KubernetesService_Deploy_Use_Latest_Version | [Preview]: Kubernetes Services should be upgraded to a non-vulnerable |Kubernetes version | The latest version of Kubernetes should be used |
+| Azure_RedisCache_DP_Use_SSL_Port | Non-SSL port must not be enabled for Redis Cache | Non-SSL port must not be enabled for Redis Cache |
+| Azure_ServiceFabric_DP_Set_Property_ClusterProtectionLevel | The ClusterProtectionLevel property must be set to EncryptAndSign for Service Fabric clusters |The ClusterProtectionLevel property must be set to EncryptAndSign for Service Fabric clusters |
+| Azure_SQLDatabase_AuthZ_Use_AAD_Admin | Use AAD Authentication for SQL Database | Enable Azure AD admin for the SQL Database |
+| Azure_SQLDatabase_DP_Enable_TDE | Enable Transparent Data Encryption on SQL databases | Enable Transparent Data Encryption on SQL databases |
+| Azure_Storage_NetSec_Restrict_Network_Access | Ensure that Firewall and Virtual Network access is granted to a minimal set of trusted origins | Ensure that Firewall and Virtual Network access is granted to a minimal set of trusted origins |
+| Azure_VirtualMachine_SI_Deploy_Data_Collection_Extension | [Preview]: Install Network data collection agents | Network traffic data collection agent should be installed on Windows/Linux virtual machines |
+
+## Frequently Asked Questions (FAQ)
+
+<br>
+
+**Even after remediating my resource, it is still showing as failing against controls in AzTS UI. The controls depends on MDC Assessment. What should I do?**
+
+**NOTE:** *Kindly make sure that the resource(s) is(are) already fixed. The controls which depends on MDC assessment could be found [here](#list-of-controls-that-depends-on-microsoft-defender-for-cloud-mdc-in-azure-tenant-security-azts).* 
+
+1. Go to **Azure Portal**.
+2. Search for **Microsoft Defender for Cloud** and **open** that.
+3. Click on **Recommendation under the General tab**, in the left side panel.
+4. Click on the **Secure Score Recommendations**.
+5. Search for the related recommendations and open it.
+6. Check the list of **unhealthy resources** to see if your resource is present in that list or not.
+7. If your resource(s) is not present in unhealthy resources list, run the scan from AzTS UI and check the status of your resource(s).
```

Comment:
Either provide screenshots or reference to MDC docs for these steps, so that it will be easy for user to follow.

Reply:
Added